mitmproxy/addons/savehar.py

```diff
@@ -207,4 +231,59 @@ def format_multidict(self, obj: _MultiDict[str, str]) -> list[dict]:
         return [{"name": k, "value": v} for k, v in obj.items(multi=True)]
 
 
-addons = [ExportHar()]
+def load(l):
```

Comment:
What are these top-level functions here doing? They are not part of the addon, so this looks like dead code?

Reply:
In the old har dump file they had it as top level functions so I kept it like that. Should I move them to a separate file?